documentation/asciidoc/computers/configuration/configuring-networking.adoc

```diff
@@ -1,183 +1,179 @@
+NOTE: Raspberry Pi devices that support the 5GHz band (Pi3B+, Pi4, CM4, Pi400), disable wireless networking until you assign a locale. To set a locale, open the `Raspberry Pi Configuration` application from the Preferences Menu, select *Localisation* and select your country.
 
 image::images/wifi2.png[wifi2]
 
-The icons on the right show whether a network is secured or not, and give an indication of its signal strength. Click the network that you want to connect to. If it is secured, a dialogue box will prompt you to enter the network key:
+The icons on the right show whether a network is secured or not and give an indication of signal strength. Click the network that you want to connect to. If the network is secured, a dialogue box will prompt you to enter the network key:
 
 image::images/key.png[key]
 
-Enter the key and click *OK*, then wait a couple of seconds. The network icon will flash briefly to show that a connection is being made. When it is ready, the icon will stop flashing and show the signal strength.
+Enter the key and click *OK*, then wait a couple of seconds. The network icon will flash briefly to show that a connection is being made. When connected, the icon will stop flashing and show the signal strength.
 
 [[wireless-networking-command-line]]
 === Using the Command Line
 
-This method is suitable if you don't have access to the graphical user interface normally used to set up a wireless LAN on the Raspberry Pi. It is particularly suitable for use with a serial console cable if you don't have access to a screen or wired Ethernet network. Note also that no additional software is required; everything you need is already included on the Raspberry Pi.
-
-==== Using raspi-config
+This guide will help you configure a wireless connection on your Raspberry Pi entirely from a terminal without using a GUI. No additional software is required; Raspberry Pi OS comes preconfigured with everything you need.
 
-The quickest way to enable wireless networking is to use the command line `raspi-config` tool.
+NOTE: This guide should work for WEP, WPA, WPA2, or WPA3 networks, but may not work for enterprise networks.
 
-`sudo raspi-config`
+==== Enable Wireless Networking
 
-Select the *Localisation Options* item from the menu, then the *Change wireless country* option. On a fresh install, for regulatory purposes, you will need to specify the country in which the device is being used. Then set the SSID of the network, and the passphrase for the network. If you do not know the SSID of the network you want to connect to, see the next section on how to list available networks prior to running `raspi-config`.
+On a fresh install, you must specify the country where you use your device.
+This allows your device to choose the correct frequency bands for 5GHz networking.
+Once you have specified a locale, you can use your Raspberry Pi's built-in wireless networking module.
```

Comment:
See earlier comment about usage of the word "locale".

Reply:
The UI explicitly now refers to it as "wireless country" I think?